cluster-up/cluster/kind-sriov/sriov-components/manifests/sriovdp-daemonset.yaml

```diff
@@ -51,79 +45,10 @@ spec:
             memory: "200Mi"
         volumeMounts:
         - name: devicesock
-          mountPath: /var/lib/kubelet/
+          mountPath: /var/lib/kubelet/device-plugins
           readOnly: false
-        - name: log
-          mountPath: /var/log
-        - name: config-volume
-          mountPath: /etc/pcidp
-        - name: device-info
-          mountPath: /var/run/k8s.cni.cncf.io/devinfo/dp
-      volumes:
-        - name: devicesock
-          hostPath:
-            path: /var/lib/kubelet/
-        - name: log
-          hostPath:
-            path: /var/log
-        - name: device-info
-          hostPath:
-            path: /var/run/k8s.cni.cncf.io/devinfo/dp
-            type: DirectoryOrCreate
-        - name: config-volume
-          configMap:
-            name: sriovdp-config
-            items:
-            - key: config.json
-              path: config.json
-
----
-apiVersion: apps/v1
-kind: DaemonSet
-metadata:
-  name: kube-sriov-device-plugin-ppc64le
```

Comment:
Could you please add reasoning for the removal of this DS?
Same for `arm64`.

Reply:
I assume the old manifest was taken from the examples of sriov-dp as is, the SR-IOV provider does not support arm.